modules/apps/depot/depot-web/src/main/java/com/liferay/depot/web/internal/info/item/provider/DepotEntryInfoItemCapabilitiesProvider.java

```diff
@@ -0,0 +1,45 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2025 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.depot.web.internal.info.item.provider;
+
+import com.liferay.depot.model.DepotEntry;
+import com.liferay.info.item.capability.InfoItemCapability;
+import com.liferay.info.item.provider.InfoItemCapabilitiesProvider;
+import com.liferay.layout.page.template.info.item.capability.DisplayPageInfoItemCapability;
+import com.liferay.portal.kernel.feature.flag.FeatureFlagManagerUtil;
+import com.liferay.portal.kernel.util.ListUtil;
+
+import java.util.Collections;
+import java.util.List;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author Marco Leo
+ */
+@Component(
+	property = "item.class.name=com.liferay.depot.model.DepotEntry",
+	service = InfoItemCapabilitiesProvider.class
+)
+public class DepotEntryInfoItemCapabilitiesProvider
+	implements InfoItemCapabilitiesProvider<DepotEntry> {
+
+	@Override
+	public List<InfoItemCapability> getInfoItemCapabilities() {
+		if (!FeatureFlagManagerUtil.isEnabled("LPD-17564")) {
```

Comment:
@adolfopa, checking the FF shouldn't be necessary with the infoPermissionProvider, but we can leave it